core/plugins/org.polarsys.capella.core.sirius.ui/src/org/polarsys/capella/core/sirius/ui/actions/AbstractNewRepresentationAction.java

```diff
@@ -40,8 +43,24 @@ protected String getDescriptionLabel(RepresentationDescription description) {
   }
 
   protected ImageDescriptor getDescriptionImageDescriptor(RepresentationDescription description) {
-    ImageDescriptor imageDescriptor = EObjectLabelProviderHelper.getImage(description.eClass(),
-        (AdapterFactoryEditingDomain) TransactionUtil.getEditingDomain(description));
+    ImageDescriptor imageDescriptor = null;
```

Comment:
Can you commonalize code with the class aside, https://github.com/eclipse/capella/blob/7379f689cab520acb859d18b2a9500a432de5d50/core/plugins/org.polarsys.capella.core.sirius.ui/src/org/polarsys/capella/core/sirius/ui/actions/OpenRepresentationsAction.java#L71-L90